### PR TITLE
updated python binding names in all pytests 

### PR DESCRIPTION
**Pull request by AadityaSuri**
### Description
updated python binding names in all tests from xx -> xx_cpp, ex: tbots -> tbots_cpp

### Testing Done

ran all simulatedTests and other tests, all running well
### Resolved Issues
Update python binding import alias #2989

### CODE REVIEW

File: src/software/thunderscope/field/simulator_layer.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import software.python_bindings as geom
+import software.python_bindings as geom_cpp
```
Comment (nimazareian) [inline src/software/thunderscope/field/simulator_layer.py:1]: If this import is not being used in this file, we should remove it

File: src/software/thunderscope/field/world_layer.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import math
 
-import software.python_bindings as geom
+import software.python_bindings as geom_cpp
```
Comment (nimazareian) [inline src/software/thunderscope/field/world_layer.py:3]: Lets keep the naming consistent as `tbots_cpp`

**Review by nimazareian — COMMENTED** [review]
Good job with making this change! Just had a couple of nits. 
Also, in the future I recommend not working directly off of your fork's master branch, and instead creating a new branch from the `upstream/master` branch for every ticket you work on. i.e.
```bash
# Go to the main repo's (upstream) master branch and pull its latest version 
git checkout upstream/master
git pull upstream master
# Create a new branch under your own fork for the specific task youre working on
git checkout -b <branch_name_related_to_ticket>
```

Also, I noticed a couple of places where we do something like `from software.python_bindings import *`. For consistency, can we also make it so we also use the `tbots_cpp` namespace there as well. E.g. Note that in this case anything that's imported (such as `SSLRefereeProtoListener`) will then need to have the prefix. 
https://github.com/UBC-Thunderbots/Software/blob/aaea6b7a9f09ea4c7706b5543de89c6aa8872456/src/software/thunderscope/robot_communication.py#L5

This should also be updated:

https://github.com/UBC-Thunderbots/Software/blob/30a4932166d2f357f65ca3e4cab88f7e3428c42a/src/software/thunderscope/thunderscope_main.py#L11

**Comment by nimazareian** [comment]
Also, I recommend updating the PR description to say "resolves #2989" so the ticket is automatically closed once you merge your PR! More keywords can be found here: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: src/software/thunderscope/field/world_layer.py (hunk 2 above)
Comment (AadityaSuri) [reply src/software/thunderscope/field/world_layer.py:3]: thanks for the review, ill make the all the suggested changes.

**Comment by nimazareian** [comment]
Closing in favour of #3021

**Pull request closed without merging**